obspy/signal/tests/test_spectral_estimation.py

```diff
@@ -932,32 +931,32 @@ def test_can_read_npz_without_pickle(self):
             PPSD.load_npz(self.example_ppsd_npz)
```

Comment:
@d-chambers Could you take a look on that code here? I have some doubts that it really does what it says.

Reply:
I think this part of the test is checking if this error is raised:
![obraz](https://user-images.githubusercontent.com/10303802/108501996-63db6580-72b2-11eb-9aae-e21693dd6cb9.png)

(sorry for screenshot, gh does not allow for commenting code that is out of the scope of a diff...)

I am not sure that this message says the truth. At that point, the data is already loaded so passing a `allow_pickle=True` does not change anything.

Reply:
so you'd want the msg to say "cannot convert back to list"? (or it could actually silently do it ?)